pkg/config/containers.conf

```diff
@@ -34,6 +34,12 @@
 #
 # apparmor_profile = "container-default"
 
+# List of additional annotation. Specified as
```

Comment:
s/annotation/annotations/

Reply:
@TomSweeneyRedHat @rhatdan if you guys agree, I can file a quick PR to address @vrothberg comments.